### PR TITLE
Setup script

### PR DESCRIPTION
**Pull request by vic**
Add a `setup.bash` script that users can execute just after clonning this repo that helps with replacing all the placeholders in template.

### CODE REVIEW

**Comment by jthegedus** [comment]
@vic This is a brilliant addition! Wish I had waited a day before starting on another plugin 😆

**Comment by imbsky** [comment]
The format style of shfmt that I always use is slightly different from the standard. If there is no problem in using it as a template, it's okay with me, what do you think?

> Use -i N to indent with a number of spaces instead of tabs. There are other formatting options - see shfmt -h. For example, to get the formatting appropriate for Google's Style guide, use shfmt -i 2 -ci.

**Comment by imbsky** [comment]
and it seems that shfmt can be configured using the editorconfig file.
https://github.com/mvdan/sh

**Comment by jthegedus** [comment]
Are you suggesting using Google's standard, or just spaces over tabs?

I'm okay with it in general as people can remove if they don't like it.

**Comment by imbsky** [comment]
No, Google standard is already enforced in the format check in CI. (I think it's a relatively reasonable style.) Well, if users don't like that, users can choose by themselves as you say, so it may not be a problem.

**Comment by vic** [comment]
> @vic This is a brilliant addition! Wish I had waited a day before starting on another plugin 😆

HAha, you can always create another one. I'm doing one for https://get-coursier.io/ and once you have that you could have asdf-managed java applications downloaded directly from public repos. And even a much better `asdf-java` might be possible too.

**Comment by vic** [comment]
> The format style of shfmt that I always use is slightly different from the standard. If there is no problem in using it as a template, it's okay with me, what do you think?
> 
> > Use -i N to indent with a number of spaces instead of tabs. There are other formatting options - see shfmt -h. For example, to get the formatting appropriate for Google's Style guide, use shfmt -i 2 -ci.

I just copied that shfmt from one of the plugins I have installed, can't even remember which one :D, but yeah.. feel free to adapt for best style. Just added you as collaborator on this repo, since I've seen you maintain a lot of the `asdf-community`, you might have a much much better opinion on what can be common between plugins.


Btw. @jthegedus, I've also implemented basic list-all and install functions, people should just have to make minimal changes if they are installing things from github releases.

File: README.md [hunk 1]
```diff
@@ -7,26 +7,7 @@ For more information about creating a plugin, see the [plugins create section of
...
-     - expand on this contributing guide as you see fit
-   - `README_TEMPLATE.md`
-     - replace `<YOUR TOOL>`
-     - replace `<YOUR NAME>`
-     - replace `<YOUR GITHUB USERNAME>`
-     - replace `<REPO>`
-     - replace the items in the `Dependencies` section
-     - rename file to README.md overwriting this file
-
+2. Clone this repo and run `bash setup.bash`
```
Comment (Stratus3D) [inline README.md]: No need for `bash setup.bash`. `./setup.bash` should be enough, assuming the executable bit is set.

**Pull request merged**